app/controllers/api/teams_controller.rb

```diff
@@ -28,6 +28,35 @@ def index
 
   private
 
+  def fetch_team
+    @team = Team.find(team_id)
+    authorize @team, :team_member?
+
+    receive_transferred_encryptables(@team) if team.personal_team?
```

Comment:
No, you said that you would like to receive the transferred encryptables directly on click on the team. So it would now decrypt all transferred encryptable with transfer_password and then encrypt with personal_team_password on click on the personal_team. No matter if team is already recrypted or not